crypto/batchverifier.go

```diff
@@ -123,11 +128,18 @@ func (b *BatchVerifier) VerifyWithFeedback() (failed []bool, err error) {
 	if b.GetNumberOfEnqueuedSignatures() == 0 {
 		return nil, nil
 	}
-	var messages = make([][]byte, b.GetNumberOfEnqueuedSignatures())
+
+	const estimatedMessageSize = 64
+	msgLengths := make([]uint64, 0, len(b.messages))
+	var messages = make([]byte, 0, b.GetNumberOfEnqueuedSignatures()*estimatedMessageSize)
```

Comment:
It's a bit odd that the line above refers to `len(b.messages)` but this line uses `b.GetNumberOfEnqueuedSignatures()`.  Makes it less obvious why this is the same thing.